.github/workflows/ui-ci.yml

```diff
@@ -57,17 +57,24 @@ jobs:
       QW_TEST_DATABASE_URL: postgres://quickwit-dev:quickwit-dev@postgres:5432/quickwit-metastore-dev
     steps:
       - uses: actions/checkout@v4
-      - uses: c-hive/gha-yarn-cache@v2
+      - uses: actions/setup-node@v3
         with:
-          directory: ./quickwit/quickwit-ui
+          node-version: 18
+          cache: "yarn"
+          cache-dependency-path: quickwit/quickwit-ui/yarn.lock
+      - name: Install rust
+        run: curl https://sh.rustup.rs -sSf | sh -s -- -y
+      - uses: arduino/setup-protoc@v2
```

Comment:
why do we need that?

Reply:
I had a failure because protoc was not installed. Let me try again.

Reply:
ah yes... we need that for building quickwit :/
the "cypress run" name made me forget that we are doing a `cargo build` and we need protoc...

Reply:
now I'm wondering how the release build works, I don't see any install of `protoc` anywhere

Reply:
ah through [`cross`](https://github.com/cross-rs/cross)! didn't know that tool!